.gitignore

```diff
@@ -11,3 +11,5 @@ project.properties
 .vscode
 .factorypath
 .DS_Store
+bankjs/development
```

Comment:
What does this folder reference to?

Reply:
was me being lazy instead of working on a different branch